deployment/ansible/roles/nyc-trees.tiler/tasks/app.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Clone project source
+  git: repo=https://github.com/azavea/nyc-trees.git
+       dest=/opt/nyc-trees
+       version="{{ tiler_deploy_branch }}"
+  when: packing
+
+- name: Ensure that tiler_home exists
+  file: path="{{ tiler_home }}"
+        state=directory
+  when: packing
+
+- name: Synchronize Tiler application into tiler_home
```

Comment:
Other references to the tiler inside a `name` attribute are lowercase. Consider making this lowercase as well for consistency.